Symfony/CS/Fixer/PSR2/MethodArgumentSpaceFixer.php

```diff
@@ -69,13 +85,46 @@ public function fixSpace(Tokens $tokens, $index)
             }
         }
 
-        // add space after comma if not exist
-        if (!$tokens[$index + 1]->isWhitespace()) {
+        $nextToken = $tokens[$index + 1];
+
+        // Two cases for fix space after comma (exclude multiline comments)
+        //  1) multiple spaces after comma
+        //  2) no space after comma
+        if ($nextToken->isWhitespace()) {
+            if ($this->isCommentLastLineToken($tokens, $index + 2)) {
+                return;
+            }
+
+            $newContent = ltrim($nextToken->getContent(), " \t");
+
+            if ('' === $newContent) {
+                $newContent = ' ';
+            }
+
+            $nextToken->setContent($newContent);
+
+            return;
+        }
+
+        if (!$this->isCommentLastLineToken($tokens, $index + 1)) {
             $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
         }
     }
 
     /**
+     * Check if last item of current line is a comment
+     *
+     * @param Tokens $tokens tokens to handle
+     * @param int    $index  index of token
+     *
+     * @return bool
+     */
+    private function isCommentLastLineToken(Tokens $tokens, $index)
+    {
+        return $tokens[$index]->isComment() && 1 === substr_count($tokens[$index + 1]->getContent(), "\n");
```

Comment:
I edited my comment to fix a typo in it. I'm talking about multiline comments, **not about T_DOC_COMMENTS**. Not all multiline comments are doc comments. Doc comments are starting with ``/**``